flaskServer/testClass.py

```diff
@@ -0,0 +1,4 @@
+class TestClass:
```

Comment:
As with TcpServer, renaming this class and the file to be more specific could improve the clarity of the code.